app/models/planet.py

```diff
@@ -0,0 +1,15 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable=False)
+    description = db.Column(db.Text, nullable=False)
+    rings = db.Column(db.Boolean, nullable=False)
+
+    def build_planet_dict(self):
```

Comment:
Prefer to leave `planet` out of the name of this function, so `build_dict`, or as we used in RT, `to_dict`. Since this is a method of `Planet`, it's a little redundant to have the class' own name in the methd. We probably will have an instance in a variable named something like `planet`, in which case, calling this would look like `planet.build_planet_dict()`. If we shorten the name, it would be `planet.build_dict()`, or `planet.to_dict()`, which already communicates that we're converting a planet to a dictionary representation.